app-5-rdf-serializer/src/main/java/org/geovistory/toolbox/streams/rdf/processors/project/ProjectOwlSameAs.java

```diff
@@ -0,0 +1,146 @@
+
+    @Inject
+    RegisterInputTopic registerInputTopic;
+
+
+    @Inject
+    OutputTopicNames outputTopicNames;
+
+    public ProjectOwlSameAs(AvroSerdes avroSerdes, RegisterInputTopic registerInputTopic, OutputTopicNames outputTopicNames) {
+        this.avroSerdes = avroSerdes;
+        this.registerInputTopic = registerInputTopic;
+        this.outputTopicNames = outputTopicNames;
+    }
+
+    public void addProcessorsStandalone() {
+        addProcessors(
+                registerInputTopic.projectStatementWithEntityStream(),
+                registerInputTopic.projectStatementWithLiteralStream()
+        );
+    }
+
+    public ProjectRdfReturnValue addProcessors(
+            KStream<ProjectStatementKey, ProjectStatementValue> projectStatementWithEntityStream,
+            KStream<ProjectStatementKey, ProjectStatementValue> projectStatementWithLiteralStream
+    ) {
+
+        /*
+         * 2) KTable of left statement
+         * 2a) FlatMap the stream to List<KeyValue<ProjectEntityKey, Boolean>>
+         */
+
+        var s1 = projectStatementWithEntityStream.flatMap(
+                (key, value) -> {
+                    List<KeyValue<ProjectEntityKey, TextWithDeleteValue>> result = new LinkedList<>();
+
+                    if (value.getStatement().getPropertyId() == 1943) {
+                        var k = new ProjectEntityKey().newBuilder().setProjectId(key.getProjectId()).setEntityId(value.getStatement().getObjectId()).build();
+                        var textWithDeleteValue = TextWithDeleteValue.newBuilder()
+                                .setText(value.getStatement().getSubjectId())
+                                .setDeleted(value.getDeleted$1())
+                                .build();
+
+                        result.add(KeyValue.pair(k, textWithDeleteValue));
+
+                    }
+
+                    return result;
+                }
+        );
+
+        // 2b) ToTable: Convert the stream to a table
+        var table1 = s1.toTable(
+                Named.as("project_statement_with_entity_stream_filtered"),
+                Materialized.with(avroSerdes.ProjectEntityKey(), avroSerdes.TextWithDeleteValue())
+        );
+
+        /*
+         * 3) KTable of right statement
+         * 3a) FlatMap the stream by List<KeyValue<ProjectEntityKey, TextWithDeleteValue>>
+         */
+        var s2 = projectStatementWithLiteralStream.flatMap(
+                (key, value) -> {
+                    List<KeyValue<ProjectEntityKey, TextWithDeleteValue>> result = new LinkedList<>();
+
+                    if (value.getStatement().getPropertyId() == 1843 && value.getStatement().getObject().getAppellation().getString().matches("^[a-z](?:[-a-z0-9\\+\\.])*:(?:\\/\\/(?:(?:%[0-9a-f][0-9a-f]|[-a-z0-9\\._~!\\$&''\\(\\)\\*\\+,;=:@])|[\\/\\?])*)?")) {
+                        var k = new ProjectEntityKey().newBuilder().setProjectId(key.getProjectId()).setEntityId(value.getStatement().getSubjectId()).build();
```

Comment:
remove the "new" keyword. newBuilder() is a static function.

Reply:
Fixed